### DISCUSSION TITLE
Server Auto Start?

### DISCUSSION THREAD

**Discussion by EthanKC** [nextgenhealthcare/connect#5053] [Q&A]
Is there a way to set the Mirth Connect Server to auto start when the device is booted up? I have it installed on Windows Server 2019 Datacenter, and it's a client owned server. I'm hoping that we don't have to start the Mirth server back up every time they need to restart their device.

**Comment by pacmano1** [comment]
It is a standard windows service.   So sure.

**Reply by pacmano1** [reply]
<img width="975" alt="image" src="https://user-images.githubusercontent.com/44065187/155419777-3f63718c-8400-4097-8f68-45457c6f276e.png">

**Reply by EthanKC** [reply]
See, I thought that would be it, but even when I have the Mirth Connect Server running, that service shows as not running. And I set it to Automatic, but it's not starting the server when I restart the device. 
<img width="889" alt="image" src="https://user-images.githubusercontent.com/100318963/155421062-71a9a92a-8d77-410a-8ad7-4a152f24c6c4.png">

**Reply by pacmano1** [reply]
Dunno in your case - it just starts for me on reboot and service manager looks correct showing the service as Running..

**Reply by tonygermano** [reply]
If you are already running the server in a shell, that will prevent the service from starting if it's using the same configuration file because the ports it needs are already in use. If you happen to be using the default Derby database for your backend (change that if you are) then those db files can also only be opened by one process at a time. Even if you aren't using Derby, connecting two instances to the same database can cause a lot of unforeseen issues (most of them handled by the commercial Advanced Clustering plugin.)

**Reply by pacmano1** [reply]
Doh!  I missed the fact that left windows was actually the server process running and not a `type mirth.log`
